core/src/test/scala/tech/beshu/ror/unit/acl/blocks/definitions/UnboundidLdapAuthenticationServiceTests.scala

```diff
@@ -124,6 +203,15 @@ class UnboundidLdapAuthenticationServiceTests
       .right.getOrElse(throw new IllegalStateException("LDAP connection problem"))
   }
 
+  private def createCircuitBreakerDecoratedSimpleAuthenticationService() = {
+    new CircuitBreakerLdapAuthenticationServiceDecorator(
+      createSimpleAuthenticationService(),
+      CircuitBreakerConfig(
+        maxFailures = Refined.unsafeApply(2),
```

Comment:
this one should work when you import `auto` (sadly I don't know how we can achieve it for our custom PositiveFiniteDuration` class)

_(We can do it in a second part of the task)_